t/ingredients_tags.t

```diff
@@ -82,6 +82,8 @@ foreach my $test_ref (@tests) {
 	my $product_ref = $test_ref->[0];
 	my $expected_tags = $test_ref->[1];
 
+	print STDERR "ingredients_text: " . $product_ref->{ingredients_text} . "\n";
```

Comment:
`print STDERR` isn't recommended in unit tests: https://stackoverflow.com/a/6162805/11963